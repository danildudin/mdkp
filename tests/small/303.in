6 3
15 33 49 71 24 31 
8 9 12 1 9 4 13
24 9 89 32 5 65 338
87 25 23 55 64 89 124
