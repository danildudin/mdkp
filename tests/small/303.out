120
2
2 3 
